test/metabase/db/schema_migrations_test/impl.clj

```diff
@@ -196,11 +197,25 @@
                   (str "'Empty' application DB is not actually empty. Found tables:\n"
                        (u/pprint-to-str tables))))))
     (log/debugf "Finding and running migrations before %s..." start-id)
-    (run-migrations-in-range! conn [1 start-id] {:inclusive-end? false})
-    (f
-     (fn migrate! []
-       (log/debugf "Finding and running migrations between %s and %s (inclusive)" start-id (or end-id "end"))
-       (run-migrations-in-range! conn [start-id end-id]))))
+    (run-migrations-in-range! conn ["v00.00-000" start-id] {:inclusive-end? false})
+    (letfn [(migrate
```

Comment:
previously , in order to run `migrate! :down` in migration tests, people have to manually call `db.setup/migrate! :down 47`.

this change makes it so that you can call downgrade with the `migrate` binding when using with `test-migrations` macro.